src/skydb.ts

```diff
@@ -124,13 +140,11 @@ export async function setJSON(
     }
   } else {
     // Assert the input is 64 bits.
```

Comment:
Looks like the comment is in the wrong spot now. However, I'd move the assertion out of the `else` clause, after the `if-else` blocks. We might as well assert the `revision` calculated in the `if`, even though it should never fail.